test/test_pipe_baseline.py

```diff
@@ -1,21 +1,53 @@
 import unittest
+import resonance.pipe as pipe
+import resonance.si as si
+import resonance.db as db
 import resonance.run as run
+import numpy as np
 
 
 class TestPipeBaseline(unittest.TestCase):
-    def check(self, si, blocks, expected):
-        code = '''
-from resonance import *
-in = input(1)
-out = pipe.baseline(in)
-createOutput(out, 'out')
-'''
+    def check(self, si, blocks, expected, beginOffset, endOffset):
+#        code = '''
+# from resonance import *
+# inputData = input(0)
+# out = pipe.baseline(inputData, beginOffset, endOffset)
+# createOutput(out, 'out')
+# '''
+        code = "from resonance import *\n"\
+               "inputData = input(0)\n"\
+               "out = pipe.baseline(inputData, %f, %f)\n"\
+               "createOutput(out, 'out')"\
+               % (beginOffset, endOffset)
 
         online = run.online([si], blocks, code)
         offline = run.offline([si], blocks, code)
 
         self.assertEqual(online, expected)
         self.assertEqual(online, offline)
 
-    def test_spatial(self):
+    def test_baseline(self):
+        w_si = si.Window(2, 20, 250)
```

Comment:
2 is a magic number which could be extracted from here and from the next line too